fft/src/KokkosFFT_Host_plans.hpp

```diff
@@ -104,25 +105,18 @@ auto _create(const ExecutionSpace& exec_space, std::unique_ptr<PlanType>& plan,
   return fft_size;
 }
 
-template <typename ExecutionSpace, typename PlanType,
+template <typename ExecutionSpace, typename PlanType, typename InfoType,
           std::enable_if_t<
               std::is_same_v<ExecutionSpace, Kokkos::DefaultHostExecutionSpace>,
               std::nullptr_t> = nullptr>
-void _destroy_plan(std::unique_ptr<PlanType>& plan) {
+void destroy_plan_and_info(std::unique_ptr<PlanType>& plan, InfoType&) {
```

Comment:
Not all changes look related to the PR. Can you do the `InfoType` change in a separate PR?